libs/extensions/angular/index.ts

```diff
@@ -0,0 +1 @@
+export * from '@kirbydesign/extensions-angular/banner';
```

Comment:
Consider leaving this file empty, as we'll want to promote importing from secondary entry points 🤞🏻 
```suggestion
// Primary entry-point which is empty. All components should
// be imported through their individual entry-points. This file is needed to
// satisfy ng_package which requires a primary entry-point.
```